api/routes/riddle.js

```diff
@@ -1,15 +1,18 @@
-const fs = require('fs');
+import fs from 'fs';
+import express from 'express';
+
+const router = express.Router();
+
 const raw_JSON = fs.readFileSync(__dirname + '/riddles.json');
 const riddles = JSON.parse(raw_JSON).riddles;
 const max = 129;
 
+router.get('/randomRiddle', (req, res) => {
```

Comment:
This would make the path for this endpoint be: `/riddle/randomRiddle` if that is where you wanted it then yeah that's fine

Reply:
I did want this, this file is where I want to write all the quick and rough test routes during dev so they won't get accidentally called during actual running. Should probably change it from '/riddle/' to '/test/'